src/gt.ts

```diff
@@ -0,0 +1,1016 @@
+const defaultOptions = [
+  {
+    name: "--version",
+    description: "Show Graphite CLI version",
+    priority: 0,
```

Comment:
You can add this property to "global" options and just add the default options to the `completionSpec.options` array removing them from subcommands. In this way all options with this property set to true will be inherited by subcommands.
```suggestion
    priority: 0,
    isPersistent: true,
```

Reply:
Alright - I didn't know about the `isPersistent` flag 👌🏼 
Changes are implemented ✅